packages/desktop-electron/window-state.ts

```diff
@@ -18,24 +27,27 @@ function loadState() {
   } catch (e) {
     console.log('Could not load window state');
   }
+
   return validateState(state);
 }
 
-function updateState(win, state) {
-  const screen = electron.screen || electron.remote.screen;
+function updateState(win: BrowserWindow, state: WindowState) {
+  const screen = electron.screen;
   const bounds = win.getBounds();
   if (!win.isMaximized() && !win.isMinimized() && !win.isFullScreen()) {
-    state.x = bounds.x;
```

Comment:
Fixed this bug, it was only saving monitor position when the window isn't maximized/fullscreen/minimized. 

This caused an issue where if you moved the maximized app over to another display it wasn't saving what display it belonged to - it would always open on primary display. 

If the display gets removed, it reverts to primary (handled in the validate function).